android/src/main/kotlin/vn/casperpas/sound_stream/SoundStreamPlugin.kt

```diff
@@ -179,8 +179,8 @@ public class SoundStreamPlugin : FlutterPlugin,
         }
     }
 
-    override fun onRequestPermissionsResult(requestCode: Int, permissions: Array<out String>?,
-                                            grantResults: IntArray?): Boolean {
+    override fun onRequestPermissionsResult(requestCode: Int, permissions: Array<out String>,
+                                            grantResults: IntArray): Boolean {
         when (requestCode) {
             audioRecordPermissionCode -> {
                 if (grantResults != null) {
```

Comment:
Could you please remove this condition? since `grantResults` will never be `null`